src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -189,6 +212,21 @@ public ClusterHealthStatus getStatus() {
         return indices;
     }
 
+    /**
+     *
+     * @return The wait time in the queue for the first pending task
+     */
+    public TimeValue getPendingTaskTimeWaitingInQueue() {
+        return pendingTaskTimeWaitingInQueue;
+    }
+
+    /**
+     * The percentage of recovery completed, should be 100% in a green system
+     */
+    public double getRecoveryPercent() {
```

Comment:
I think we should to `activeShardsPercent` . I think recovery is misleading as it isn't clear whether you should count shards that haven't even starting recovering (aren't initializing) or are not found. It will also be more inline with the rest of the numbers we expose.